src/css/showHide.js

```diff
@@ -4,6 +4,23 @@ define( [
 	"../css/var/isHidden"
 ], function( jQuery, dataPriv, isHidden ) {
 
+var defaultDisplayMap = {};
+
+function setDefaultDisplay( elem ) {
+	var doc = elem.ownerDocument,
+		nodeName = elem.nodeName,
+		temp = doc.body.appendChild( doc.createElement( nodeName ) ),
+		display = jQuery.css( temp, "display" );
+
+	temp.parentNode.removeChild( temp );
+
+	if ( display === "none" ) {
+		display = "block";
```

Comment:
Initially, i have wrote "outrages use-case" there - https://github.com/jquery/jquery/pull/2810#discussion_r49628474, but yeah totally, they really are like that i suppose, my biggest concern is hidden iframe though, which also shouldn't be that common.